### PR TITLE
azurerm_netapp_volume - support for the zone property

### PR DESCRIPTION
**Pull request by paulomarquesc**
This PR implements support for availability zones on ANF volumes.

### CODE REVIEW

File: internal/services/netapp/netapp_volume_resource.go [hunk 1]
```diff
@@ -503,6 +514,16 @@ func resourceNetAppVolumeUpdate(d *pluginsdk.ResourceData, meta interface{}) err
...
+	if d.HasChange("zones") {
+		return fmt.Errorf("zone changes are not supported after volume is already created, %s", id)
+	}
+
+	if d.HasChange("storage_quota_in_gb") {
+		shouldUpdate = true
+		storageQuotaInBytes := int64(d.Get("storage_quota_in_gb").(int) * 1073741824)
+		update.Properties.UsageThreshold = utils.Int64(storageQuotaInBytes)
+	}
+
```
Comment (favoretti) [inline internal/services/netapp/netapp_volume_resource.go:526]:
```suggestion
```
This is duped by accident? Same block is below.

**Review by favoretti — CHANGES_REQUESTED** [review]
Thanks @paulomarquesc, one minor comment from my side, pending maintainers' review.

**Review by katbyte — APPROVED** [review]
Thanks @paulomarquesc - this looks good aside from mine and @favoretti's comments! should be good to merge once addressed

File: internal/services/netapp/netapp_volume_resource.go (hunk 1 above)
Comment (paulomarquesc) [reply internal/services/netapp/netapp_volume_resource.go:526]: Thanks for good catch @favoretti , seems a conflict merge issue. Fixing it.

**Pull request merged**